src/dev/Playground.tsx

```diff
@@ -10,6 +10,8 @@ export default class Example extends Component {
     showNavigation: true,
     config: config.gentle,
     goToSlideDelay: 200,
+    autoplay: true,
+    interval: 2,
```

Comment:
Maybe we can just use `interval` here and default it to `null` or `0`, this would simplify the API and prevent users from accidentally setting inconsistent prop values.